lispusers/CALENDAR

```diff
@@ -1165,7 +1167,7 @@ Copyright (c) 1985-1990 by Xerox Corporation.
                                                                (CHARACTER 13)
```

Comment:
Is `(CHARACTER 13)` what it should be doing here?  I would have thought since we have EOLCHARCODE (from BOOTSTRAP) that (CHARACTER EOLCHARCODE) would be  clearer about its intention.   Perhaps that's a change for another time though.